msg/RoverDifferentialStatus.msg

```diff
@@ -1,8 +1,10 @@
 uint64 timestamp # time since system start (microseconds)
 
 float32 actual_speed	       # [m/s] Actual forward speed of the rover
-float32 desired_yaw_rate_deg_s # [deg/s] Desired yaw rate
+float32 actual_yaw_deg         # [deg] Actual yaw of the rover
+float32 desired_yaw_rate_deg_s # [deg/s] Desired yaw rate (This is deviates from the yaw_rate_setpoint if the yaw setpoint is used)
```

Comment:
```suggestion
float32 desired_yaw_rate_deg_s # [deg/s] Desired yaw rate (deviates from the yaw_rate_setpoint if the yaw setpoint is used)
```

Reply:
I don't fully get it by just looking at the comments of this message

Reply:
If the yaw setpoint is used, then the desired yaw rate is calculated using the yaw PID. 
This calculated yaw rate is then used in the closed loop yaw rate control instead of the desired_yaw_rate setpoint.
So by using the yaw_setpoint the desired_yaw_rate_setpoint and the yaw rate the controller actually tries to achieve are different (desired_yaw_rate_setpoint is ignored). 
I agree that it is quite complicated and not clear from the comment, I'll think of a better way to explain it.

Reply:
Tried to clarify with [this](https://github.com/PX4/PX4-Autopilot/pull/23629/commits/2c76eb214a78a493dc9e1aac76e988cf7a0ec2b3#diff-fdc56dd532fa90878add2ff88f33a2a203c15e43afa376b1105c7a466e684366R5) and [this](https://github.com/PX4/PX4-Autopilot/pull/23629/commits/2c76eb214a78a493dc9e1aac76e988cf7a0ec2b3#diff-b8337121cc9d7e2c5d40d58288cfc355cd1ea2707d18bce8379cec49c4c3405cR6).